docs/6.0. Upgrade Path.md

```diff
@@ -8,10 +8,12 @@ API versioning is specified in the [APIs](2.0.%20APIs.md) documentation, with pr
 
 ## Requirements for Nodes (Node APIs)
 
-Implementers of the Node API must support at least one API version, and may support more than one at a time. Note however that a Node must only perform interactions with a Registration API at a single version. Nodes implementing multiple API versions may provide a user-configurable choice for which API version to register and/or query using.
+Implementers of the Node API must support at least one API version, and may support more than one at a time. Note however that a Node must only perform interactions with a Registration API at a single version. Nodes implementing multiple API versions may provide a user-configurable choice for which API version to register and/or query using. Otherwise it is expected that the Node will select a Registration API which supports the highest API version which the Node also supports. The API version should be selected before considering the Registration API's configured \'priority\'.
```

Comment:
Do you want to use "MUST"/"MAY" in this section rather than "must"/"may"?

Reply:
Yes, I should probably fix that whilst editing. There are likely other cases of this in the docs which need resolving too.